code/game/machinery/doors/simple.dm

```diff
@@ -73,13 +70,15 @@
 /obj/machinery/door/unpowered/simple/close(var/forced = 0)
 	if(!can_close(forced))
 		return
-	playsound(src.loc, material.dooropen_noise, 100, 1)
+	if(material)
```

Comment:
not sure if there's any point checking for material considering it frekaing qdels self if it hasn't got one